components/LeagueCard/LeagueCard.test.tsx

```diff
@@ -85,4 +86,27 @@ describe('LeagueCard with custom league logo', () => {
     expect(leagueCardTitle).toBeInTheDocument();
     expect(leagueCardTitle).toHaveTextContent('League 1');
   });
+
+  it('should apply hover effect on mouse hover', () => {
+    render(
+      <Link
+        href="/league/all"
+        data-testid="customLinkCard"
+        className="customLinkCard hover:bg-zinc-800"
+      >
+        <LeagueCard
+          href="/leagues"
+          isEliminated={true}
+          survivors={11}
+          title="League 1"
+          totalPlayers={12}
+        />
+        ,
```

Comment:
Re: the comma, saving the file added the comma, so I believe it's ok to add it in. 

Could you elaborate some more on your second question please? I assume I would have to render the LeagueCard Component to be able to test the hover since the effect occurs when hovering over the LeagueCard component. 

Please let me know what your thoughts are. Thanks!